examples/src/app.module.css

```
@@ -12,6 +12,42 @@ h5 {
   margin: 0;
 }
 
+body {
+  /* stylelint-disable */
+  font-family:
+    system,
+    -apple-system,
+    '.SFNSText-Regular',
+    'San Francisco',
+    'Roboto',
+    'Segoe UI',
+    'Helvetica Neue',
+    'Lucida Grande',
+    sans-serif;
+  /* stylelint-enable */
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 40px 0;
```

Please increase the padding so mobile browsers do no crop it until the browser bar is collapsed:

```suggestion
  padding: 56px 0;
```